README.md

```diff
@@ -47,3 +47,6 @@ and remotely.
 
 - Users should be able to provide feedback after using any facility.
 - Feedback submissions should be visible to the apartment manager/owner for review and improvement.
+
+
+# This is a test line
```

Comment:
Remove these and just add an extra line instead, so that i can test if my approval further carries the process as desired.

Reply:
but will that be considered as a change?